lib/src/actions/get_byte_code.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:js_interop';
+
+import 'package:wagmi_flutter_web/src/js/wagmi.js.dart';
+
+/// [Documentation API](https://wagmi.sh/core/api/actions/getByteCode)
+class GetByteCodeParameters {
+  GetByteCodeParameters({
+    required this.address,
+    this.blockNumber,
+    this.blockTag,
+    this.chainId,
+  });
+
+  final String address;
+  final BigInt? blockNumber;
+  final String? blockTag;
+  final int? chainId;
+
+  JSGetByteCodeParameters get toJS {
+    final jsBlockParams = JSGetByteCodeParameters(
+      address: address.toJS,
+    );
+    if (chainId != null) {
```

Comment:
I think those `if` are useless.

This is equivalent :
```dart
return JSGetByteCodeParameters(
  address: address.toJS,
  chainId: chainId?.toJS,
  blockNumber: blockNumber?.toJS,
  blockTag: blockTag?.toJS,
)
```

Reply:
After testing my solution, I get it doesn't work because Dart doesn't differenciates `null` and `undefined` types.